python/iotivity.py

```diff
@@ -1157,7 +1158,7 @@ def offboard_all_owned(self):
 
             run_count = 0
             result = False
-            while run_count < 5 and not result: 
+            while run_count < 1 and not result: 
```

Comment:
No need for while, this will now always only run once

Reply:
Thanks for that!